Option to hide description in the header

chsxf opened the pull request:
Added option to hide the description in the header. The reclaimed space can help display other things such as stats or custom links

chsxf committed:
Option to hide description in the header

godalming123 reviewed (changes requested):
good job although you should add a mention to the configuration in `readme.md`

davorpa commented on _layouts/default.html:
```diff
@@ -21,7 +21,9 @@ <h1><a href="{{ "/" | absolute_url }}">{{ site.title | default: site.github.repo
           <img src="{{site.logo | relative_url}}" alt="Logo" />
         {% endif %}
 
+        {% if site.hide_description_in_side_bar != true %}
```
I'd opt with to avoid to overcharge configuration with a merelly boolean applied over `site.description`

```suggestion
        {% if site.description != false %}
```

davorpa replied:
Resolved at [`d3fd543` (#132)](https://github.com/pages-themes/minimal/pull/132/commits/d3fd543dd08c5ed2a83287e85b2fc3c17f10abb1)

davorpa commented on _layouts/default.html:
```diff
@@ -21,7 +21,9 @@ <h1><a href="{{ "/" | absolute_url }}">{{ site.title | default: site.github.repo
           <img src="{{site.logo | relative_url}}" alt="Logo" />
         {% endif %}
 
+        {% if site.hide_description_in_side_bar != true %}
```
Or thinking in the rest of themes and preserving backwards compatibility...

```suggestion
        {% assign hide_description = site.hide_description | default: false %}
        {% if site.hide_description %}
```

davorpa replied:
Applied in [`d3fd543` (#132)](https://github.com/pages-themes/minimal/pull/132/commits/d3fd543dd08c5ed2a83287e85b2fc3c17f10abb1)

chsxf committed:
fixup! Option to hide description in the header

chsxf commented:
Based on your comments, I've added instruction to the README.md file and changed the setting to a "show" instead of a "hide" (like "show_downloads") but with a default value to true.
